test/JsonApiDotNetCoreMongoDbExampleTests/IntegrationTests/ReadWrite/Updating/Relationships/UpdateToOneRelationshipTests.cs

```diff
@@ -0,0 +1,91 @@
+using System.Net;
+using System.Threading.Tasks;
+using FluentAssertions;
+using JsonApiDotNetCore.Serialization.Objects;
+using Xunit;
+
+namespace JsonApiDotNetCoreMongoDbExampleTests.IntegrationTests.ReadWrite.Updating.Relationships
+{
+    public sealed class UpdateToOneRelationshipTests
+        : IClassFixture<IntegrationTestContext<TestableStartup>>
+    {
+        private readonly IntegrationTestContext<TestableStartup> _testContext;
+
+        public UpdateToOneRelationshipTests(IntegrationTestContext<TestableStartup> testContext)
+        {
+            _testContext = testContext;
+        }
+        
+        [Fact]
+        public async Task Cannot_replace_OneToOne_relationship()
```

Comment:
The distinction between OneToOne and ManyToOne is not relevant here, so pick one of these two tests, rename to Cannot_replace_relationship and remove the other one.